### PR TITLE
frontend: preselect backup when there's only 1 & show message when there's more

### PR DESCRIPTION
**Pull request by shonsirsha**
Preselect backup when there's only 1, and show message of "Please select backup to continue" when there are more than 1 backups available.

[Preview]:

ONLY_1

### CODE REVIEW

File: frontends/web/src/routes/device/bitbox02/setup/restore.tsx [hunk 1]
```diff
@@ -43,12 +45,15 @@ export const RestoreFromSDCardBackup = ({
       width="700px">
       <ViewHeader title={t('backup.restore.confirmTitle')} />
       <ViewContent>
+        {showSubtitleText ? <p className="m-none">{t('backup.restore.subtitle')}</p> : null}
```
[the remark this answers is not in the document]
Comment (thisconnect) [reply frontends/web/src/routes/device/bitbox02/setup/restore.tsx:48]: An alternative could be that this is rendered by BackupsV2 component itself, returning a  document fragment that conditionally includes this paragraph. 

Condition could be showRadio === true, as in this case the user is expected to selected one, that could be simpler WDYT? No strong opinion just wanted to discuss.

File: frontends/web/src/routes/device/bitbox02/setup/restore.tsx [hunk 2]
```diff
@@ -48,7 +48,8 @@ export const RestoreFromSDCardBackup = ({
           showRestore={true}
           showRadio={true}
           onSelectBackup={onSelectBackup}
-          onRestoreBackup={onRestoreBackup}>
+          onRestoreBackup={onRestoreBackup}
+        >
```
Comment (thisconnect) [inline frontends/web/src/routes/device/bitbox02/setup/restore.tsx:52]: Nit: revert this change?

**Review by thisconnect — APPROVED** [review]
tested f30a1ea9d965ff4f6345e765d2ca63deb678d497 LGTM 👍 
with small nit.

File: frontends/web/src/routes/device/bitbox02/setup/restore.tsx (hunk 2 above)
Comment (thisconnect) [reply frontends/web/src/routes/device/bitbox02/setup/restore.tsx:52]: ack 👍

**Comment by benma** [comment]
A CHANGELOG entry would be nice :smile:
